### DISCUSSION TITLE
Synchronization

### DISCUSSION THREAD

**Discussion by mhaebig** [marcusolsson/obsidian-projects#354] [Help] [answered]
Hi, 
is my observation correct: projects is not synced across devices (MacBook/Windows laptop), but I would have to set it up separately on each device?
Thanks
Manfred

**Comment by marcusolsson** [comment, marked as answer]
Are you using Obsidian Sync? The plugin stores project configuration in the `.obsidian` folder in your vault. By default though, Obsidian Sync doesn't sync vault configuration, but I believe you can configure it under the settings for Sync.

**Reply by mhaebig** [reply]
Hi, thx for your hint. I’ll try that and come back with an answer. Best, Manfred

**Reply by mhaebig** [reply]
I checked and all settings are toggled on (see attached screenshots). Nevertheless, the configuration is not synced across my computers.
[Sync_Settings.pdf](https://github.com/marcusolsson/obsidian-projects/files/10421506/Sync_Settings.pdf)

**Reply by mhaebig** [reply]
Happy End of Story:

after having also talked to a support person of Obsidian, I did the following:

-	fully sync all devices (2 laptops, 2 iPhones, 1 iPad)
-	toggle on the vault sync options in the iPhones (which were the only devices that had them not toggled on)

And now everything works.

Learning: the sync options must be turned on in ALL  the synced devices.

**Reply by ChristopherWSmith** [reply]
Adding to this as I just ran into the same issue, but my settings were already toggled on for all of my devices - I turned off the toggle for my mobile device, closed out of Obsidian on all my other devices, turned the toggle back on on mobile, and the settings synced over to all my devices.

**Reply by jheg** [reply]
This worked for me too. ^ 

I have two laptops and an iphone. Once I went in and enabled on the iPhone so all three we the same, closed and reopened them all it picked up my projects.
